src/Symfony/Bridge/Twig/Command/LintCommand.php

```diff
@@ -219,10 +236,14 @@ private function displayJson(OutputInterface $output, array $filesInfo)
         return min($errors, 1);
     }
 
-    private function renderException(OutputInterface $output, string $template, Error $exception, string $file = null)
+    private function renderException(OutputInterface $output, string $template, Error $exception, string $file = null, ?GithubActionReporter $githubReporter = null)
     {
         $line = $exception->getTemplateLine();
 
+        if ($githubReporter) {
+            $githubReporter->error($exception->getRawMessage(), $file, $line);
```

Comment:
the linter reports the line as `<= 0` when unknown (see below where it is handled). This should be passed as `null` to GithubActionReporter.